components/tidb_query/src/rpn_expr/impl_miscellaneous.rs

```diff
@@ -25,6 +25,42 @@ pub fn is_ipv4_compat(addr: &Option<Bytes>) -> Result<Option<i64>> {
     }
 }
 
+#[rpn_fn]
+#[inline]
+pub fn inet_aton(addr: &Option<Bytes>) -> Result<Option<i64>> {
+    Ok(addr
```

Comment:
Since the implementation is not trivial, I think it would be better to extract them to a common util function, and call that function in both vectorized and non-vectorized functions.

Reply:
hey, I made some changes according to your suggestion, but I'm not sure I did it right. PTAL :)